src/js/providers/video-actions-mixin.js

```diff
@@ -21,20 +22,24 @@ const VideoActionsMixin = {
         if (!width || !height || !this.video.videoWidth || !this.video.videoHeight) {
             return false;
         }
+        const _videotag = this.video;
+        const styles = {
+            objectFit: '',
+            width: '',
```

Comment:
Are there any side effects of using `''` instead of `null` for width and height?

Reply:
This doesn't; have a few more things to fix, but have made the update for my next commit.

Reply:
```js
function _styleValue(property, value) {
    if (value === '' || value === undefined || value === null) {
        return '';
    }
```